src/nester/nester_log.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module handles Nester's logging functionality. Nester logs all projects created with it unless the --no-log flag ist set when calling nester create
+"""
+
+import logging
+import re
+import sys
+from pathlib import Path
+
+
+class ProjectLogFormatter(logging.Formatter):
+    """
+    Create custom formatter class to format log
+    """
+
+    def format(self, record):
+        record = self.process(record)
+        return super().format(record)
+
+    def process(self, record):
+        record.projectname = getattr(record, "projectname", "N/A")
+        record.programming_language = getattr(record, "programming_language", "N/A")
+        record.location = getattr(record, "location", "N/A")
+        return record
+
+
+logger = logging.getLogger("nester")
+logger.setLevel(logging.INFO)
+
+log_file_path = Path.home() / "nester.log"
```

Comment:
I think that if we go this way and write into a file in the home directory we should at least make it a hidden file

Reply:
Agreed, Fixed